src/pages/community/patterns/create-flows/index.mdx

```diff
@@ -3,367 +3,440 @@ title: Create flows
-</Column>
-</Row>
+#### Don't:
 
-## Pre-create
+- Don't use the Full-page option when the product or service is usable before 
+  the current creation flow is initiated.
+- For IBM Public Cloud, do not use this when provisioning a service directly
+  from the Catalog, or when creating a non-billable resource that could happen
+  in a side panel/in context.
 
-A pre-create step occurs before the create flow starts. A common pre-create step
-is **Create from template**.
+## Creation
 
-A pre-create step is usually combined with a
-[full-page create flow](#full-page). It can also be used in combination with a
-[side panel create flow](#full-page). A pre-create step can be used in
-combination with a single-step or multistep create flow. A multistep flow starts
-after the pre-create step is complete.
+### Behaviors, structure and functionality
 
-Consider reminding users of the option they choose in pre-create in the rest of
-the create flow.
+A create flow is usually started when the user clicks on a "New `asset type`"
+button. The button text should include the type of asset and a plus icon. For
+example, "New project" or "New user". In many cases the button will be primary.
+However, there should be only one primary button in any page section. When there
+is already a primary button, a tertiary or ghost button should be used. Today,
+guidance around button types does not specify when to use tertiary versus ghost
+buttons. This guideline will be updated as we get more clarity.
 
-<Row>
-<Column colLg={8}>
+After clicking on the new button, the user then fills out a form in one of the
+above formats with one step or multiple steps. With multiple steps, the "Back"
+button is disabled in the first step. In the last step, the text of the primary
+button changes from "Next" to "Create". The "Next" and "Create" buttons should
+be disabled until the user has entered all required fields.
 
-![Example of a pre-create view](images/Precreate1.png)
+The Modal, Side panel, Narrow tearsheet and Wide tearsheet components in a
+create flow will not have a close icon in the top right. This is to enforce
+the user's decision to exit the create flow by clicking on the word "Cancel".
 
-<Caption>Example of a pre-create view</Caption>
+When the user clicks "Create" there should be an indication of loading or
+submitting. See Carbon's
+[Loading](https://www.carbondesignsystem.com/patterns/loading-pattern#loading-indicators)
+pattern. Also, see specific guidance for
+[loading in the Modal component](https://www.carbondesignsystem.com/components/modal/usage#loading).
+Once the loading is complete, the user should see a success or error
+notification. For success, one option is to open the newly created asset or the
+asset's details page. When the user stays on the same page, show a success
+banner notification. For errors, you must show an
+[error notification](https://www.carbondesignsystem.com/components/notification/usage/).
 
-</Column>
-</Row>
+### Other use cases
```

Comment:
We normally call these things "Related" so to keep the specification "Related use cases"